documentation/modules/proc-configuring-kafka-user.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-user-operator.adoc
+
+[id='proc-configuring-kafka-user-{context}']
+= Configuring a Kafka user
+
+Use the properties of the `KafkaUser` resource to configure a Kafka user.
+
+You can use `kubectl apply` to create or modify users, and `kubectl delete` to delete existing users.
+
+* `kubectl apply -f _<user-config-file>_`
+* `kubectl delete KafkaUser _<user-name>_`
+
+When you create a user, you must configure the authentication and authorization mechanisms required when connecting to Kafka brokers using a listener.
```

Comment:
I think this is confusing and the reader might not understand what he should do. Eben I'm not completely sure what does this refer to. Does it refer to the KAfkaUser `authentication` section?

Reply:
Is this clearer:

> When you create a Kafka user, you configure the `spec` to define the `authentication` and `authorization` mechanisms required when connecting to Kafka brokers using a listener.

Reply:
Sorry, no. It isn't. What are you trying to say? that the `KafkaUser.spec.authntication` should match `Kafka.spec.kafka.listeners.*.authentication`? And that `Kafka.spec.kafka.authorization` needs to be configured to use `KafkaUser.spec.authorization`? Then we should probably list this. I do not think this is really clear from it.

Reply:
Okay. How about:

> When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent `Kafka` configuration:
> 
> * `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners.*.authentication`
> * `KafkaUser.spec.authorization` matches `Kafka.spec.kafka.authorization`